en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/observability/prometheus-metric-support.md

```diff
@@ -0,0 +1,190 @@
+# Prometheus Metric Support
+
+With the integration of Prometheus metrics, monitoring the performance and health of Choreo Connect router, adapter, and enforcer components has never been easier. This enhanced observability empowers users to proactively identify and address issues for smoother operations. This feature is available starting from Choreo Connect version 1.2.0.2.
```

Comment:
Let's add a warning banner as in here to say that, this feature is only available in U2 updates

https://apim.docs.wso2.com/en/4.1.0/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-graphql-api-in-choreo-connect/

Reply:
Fixed in [5cda688](https://github.com/wso2/docs-apim/pull/7284/commits/5cda688c078f4ba5192a1d283785704cb61ce00c)